templates/zerver/help/create-a-stream.md

```diff
@@ -12,29 +15,45 @@ Streams can be either:
 
     ![Streams Overview](/static/images/help/streams-overview.png)
 
-    If you do not see the **+** button, it is probably because your organization's administrators have disabled stream creation for ordinary users. If that's the case, then you need to ask them to make creating streams available to other users, or you will have to ask an administrator for help creating each particular stream.
+    If you do not see the **+** button, it is probably because your
+    organization's administrators have disabled stream creation for
+    ordinary users. If that's the case, then you need to ask them to
+    make creating streams available to other users, or you will have
+    to ask an administrator for help creating each particular stream.
 
 4. After clicking the **+** button, the right side of the stream browser,
 labeled **Add New Stream**, will now display options for creating a stream.
 
     ![Streams Modal](/static/images/help/stream-modal.png)
 
-5. Enter the title of your stream in the **Stream name** input. If you designate
-your stream as public, the **Stream name** will be displayed under the Streams
-on the left sidebar. **Stream name** must be unique to all other streams and can
-be no longer than 60 characters.
+5. Enter the title of your stream in the **Stream name** input.
+If you designate your stream as public, the **Stream name** will be
+displayed under the Streams on the left sidebar. **Stream name** must be
+unique to all other streams and can be no longer than 60 characters.
 
     * You can optionally enter a brief description of your stream in
```

Comment:
"You can optionally enter a brief description..." should be transformed into a `!!! tip ""`